### DISCUSSION TITLE
Middle scroll

### DISCUSSION THREAD

**Discussion by Tetuas** [KadeArchive/Kade-Engine#870] [Ideas]
How about adding middle scroll? Its a nice idea so you don't have to move your neck if you have a big screen or so just to look at it better, maybe experiment with custom GUI? Just a idea you don't have to do it. That's all I got, bye.

**Comment by puyoxyz** [comment]
this would probably fit into the improved customize gameplay option

**Comment by isakube** [comment]
maybe sidescroll could be a thing too? lol

**Comment by SuperSpinDashGamer39** [comment]
yeah that would be a quality of life feature

**Reply by guizinhomartinez** [reply]
you can do that with modchart but the the enemy notes dont appear
here the code:

function start (song)
    showOnlyStrums = true -- remove all hud elements besides notes and strums
    for i=0,3 do -- fade out the first 4 receptors (the ai receptors)
        tweenFadeIn(i,0,0.6)
    end
    for i = 4, 7 do -- go to the center
        tweenPosXAngle(i, _G['defaultStrum'..i..'X'] - 275,getActorAngle(i) + 360, 0.6, 'setDefault')
    end
end

function setDefault(id)
    _G['defaultStrum'..id..'X'] = getActorX(id)
end



(btw you gotta put the song you want to have this in story menu, if it is alredy then just ignore this)
(and the healthbar dont appear but you can change this by renaming showOnlyStrums to strumLine1Visible)

**Comment by ACardboardBox1** [comment]
thinking about doing a pr for this
